com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -37,8 +60,11 @@ private static class Profiling
         internal RenderingMode actualRenderingMode { get { return GL.wireframe || m_DeferredLights == null || !m_DeferredLights.IsRuntimeSupportedThisFrame() || m_DeferredLights.IsOverlay ? RenderingMode.Forward : this.renderingMode; } }
         internal bool accurateGbufferNormals { get { return m_DeferredLights != null ? m_DeferredLights.AccurateGbufferNormals : false; } }
         internal bool usesRenderPass;
+        internal DepthPrepassMode depthPrepassMode { get { return m_DepthPrepassMode; } }
+        internal DepthPrimingMode depthPrimingMode { get { return m_DepthPrimingMode; } }
```

Comment:
Do we think users would want to change this at runtime? Maybe if so we could expose it.

Reply:
It wouldn't be super useful but there's no conceptual problems with it. Exposing it only required a conditional copy pass in the constructor to become independent of the depth priming mode at construction time.

Reply:
Ok. We can expose if they ask. Resolving conversation.